content/en/real_user_monitoring/browser/advanced_configuration.md

```diff
@@ -793,6 +841,108 @@ acceptCookieBannerButton.addEventListener('click', () => {
 {{% /tab %}}
 {{< /tabs >}}
 
+## View context
+
+Starting with [version 5.28.0][21], the contexts of view events are modifiable. The context adds to the current view only and populate its children events (such as `action`, `error` and `timing`) with `startView`, `setViewContext` and `setViewContextProperty`.
```

Comment:
```suggestion
Starting with [version 5.28.0][21], the context of view events is modifiable. Context can be added to the current view only, and populates its child events (such as `action`, `error`, and `timing`) with `startView`, `setViewContext`, and `setViewContextProperty` functions.
```

Reply:
Attempted to clarify this sentence, let me know if any of the suggestions above are inaccurate.